src/DotVVM.Framework/Compilation/Parser/Binding/Parser/BindingParser.cs

```diff
@@ -883,6 +898,94 @@ private static string ParseStringLiteral(string text, out string? error)
             return sb.ToString();
         }
 
+        private static (string, List<BindingParserNode>) ParseInterpolatedString(string text, out string? error)
+        {
+            text = ParseStringLiteral(text, out error).Substring(1);
```

Comment:
You are parsing it as a standard string literal while it may actually contain nested expressions which may also contain `"`.